noir/acvm-repo/acvm/src/pwg/blackbox/hash.rs

```diff
@@ -86,3 +86,82 @@ fn write_digest_to_outputs(
 
     Ok(())
 }
+
+const ROUNDS: usize = 24;
+
+const RC: [u64; ROUNDS] = [
+    1u64,
+    0x8082u64,
+    0x800000000000808au64,
+    0x8000000080008000u64,
+    0x808bu64,
+    0x80000001u64,
+    0x8000000080008081u64,
+    0x8000000000008009u64,
+    0x8au64,
+    0x88u64,
+    0x80008009u64,
+    0x8000000au64,
+    0x8000808bu64,
+    0x800000000000008bu64,
+    0x8000000000008089u64,
+    0x8000000000008003u64,
+    0x8000000000008002u64,
+    0x8000000000000080u64,
+    0x800au64,
+    0x800000008000000au64,
+    0x8000000080008081u64,
+    0x8000000000008080u64,
+    0x80000001u64,
+    0x8000000080008008u64,
+];
+
+const RHO: [u32; 24] =
```

Comment:
Leaving this for now, we should probably just use https://docs.rs/keccakf/latest/keccakf/ in a later PR or something that has keccak and keccakf